### ISSUE TITLE
Alien microbes do not show up in the virus list.

### ISSUE THREAD

**Issue by WJohn** [tgstation/-tg-station#8183]
This is problematic when checking if someone is infected. If a larva is supposed to show up in their contents I can't seem to see that either.

This has been the case for a loooooooong time.

**Comment by RemieRichards** [comment]
Alien embryo's aren't a virus, they haven't been for a long time.
The Alien Embryo is the same object you pull out during the surgery, it just sits in the mob's contents list until a certain timer has counted down, gibs the human, spawns a larva, grabs a host, then deletes itself.

**Comment by WJohn** [comment]
That's strange, I've never seen it in the contents before. If it's confirmed to work then I'm probably just blind and this can be closed.

**Comment by RemieRichards** [comment]
@WJohn Yesterday or the day before I was part of a natural (as in caused by the players) event, where we moved to the white shuttle and left.

Eventually I was requested to badmin it up a little bit so of course a facehugger or 2 (about 6) got left around the place, but sadly none popped (stupid surgery-competent crew) so I was watching the embryo's in the contents lists of the crew for quite a while.

tl;dr I seen it with my own eyes lord!

**Comment by WJohn** [comment]
Alright, good to know.
